pkg/controller/hostpathprovisioner/prometheus.go

```diff
@@ -439,7 +439,7 @@ func (r *ReconcileHostPathProvisioner) checkPrometheusUsed() (bool, error) {
 			// prometheus not deployed
 			return false, nil
 		}
-		return false, err
+		return false, nil
```

Comment:
I guess this is not just about SCCs
my worry with this is that we miss isErrCacheNotStarted which can cause a watch to not be set up..

I also see we ignore transient errors here in calls to checkPrometheusUsed so that
may be a problem